README.md

```diff
@@ -68,6 +68,21 @@ sudo snap install clockify-cli
 go get -u github.com/lucassabreu/clockify-cli
 ```
 
+> Starting in Go 1.17, installing executables with go get is deprecated. go install may be used instead. Read more about it in <https://go.dev/doc/go-get-install-deprecation>
+
+#### Using `go install`
+
+```sh
+go install github.com/lucassabreu/clockify-cli/cmd/clockify-cli@latest
+```
+
+The installed application for a default `go` installation should be located on `~/go/bin/clockify-cli`. You can move it or symlink it to `/usr/local/bin` so it appears on your path. You will need `root` privileges.
+
+```sh
+ln -s ~/go/bin/clockify-cli /usr/local/bin/clockify-cli
+```
```

Comment:
```suggestion
The installed application for a default `go` installation should be located on  your [$GOBIN path][go-envs]. You can add `$GOBIN` to your `$PATH`, or move it to a directory listed on your `$PATH` (e.g.: `/usr/local/bin`).

[go-envs]: https://pkg.go.dev/cmd/go#hdr-Environment_variables

```

Reply:
about the symlinking... i think go users would already have `$GOBIN` on their `$PATH` so they wouldn't need to symlink...

I have seen `ln` change argument order and flags on same distros... so i think is better not to recommend it, if the user knows about `ln` and `symlinking` they don't need the line normally.

Reply:
I have go installed (following instructions from https://go.dev/doc/install) and I don't have `$GOBIN` in my env vars, don't know why. About the argument order, I can't say it happens in my case (linux mint 21.1 and `go version go1.20.1 linux/amd64`). I agree with the changes you've done. Thanks for replying!